docs/user/azure/limits.md

```diff
@@ -0,0 +1,124 @@
+# Limits
+
+You can find a comprehensive list of the default Azure service limits published here:
+
+[Azure Service Limits][service-limits]
+
+Below, we'll identify OpenShift cluster needs and how those impact some of those limits.
+
+## VNet
+
+Each cluster creates its own VNet. The default limit of VNets per regions is 1000 and will allow 1000 clusters. To 
+have more than 1000 clusters, you will need to increase this limit. The default installation creates a single VNet.
+
+## Network Interfaces
+
+The default installation creates 6 network interfaces. The default limit per region is 65536. Additional network interfaces are
+created for additional machines and load balancers created by cluster usage and deployed workloads.
+
+## Public and Private IP Addresses
+
+By default, the installer distributes control-plane and compute machines across [all availability zones within a region][availability-zones] to provision the cluster in a highly available configuration.
+Please see [this map][az-map] for a current region map with availability zone count.
+We recommend selecting regions with 3 or more availability zones.
+You can [provide an install-config](../overview.md#multiple-invocations) to [configure](customization.md) the installer to use specific zones to override the defaults.
+
+The installer creates an internal load balancer and two subnets within the VNet. There is a subnet for the control plane nodes and there is a subnet for the compute nodes. The internal load balancer balances off the compute nodes while an external load balancer with a public IP address balances out the control plane nodes.
+
+## Network Security Groups
+
+Each cluster creates network security groups for the control plane and for compute nodes. The default limit of 5000
```

Comment:
the network security groups in Azure are attached to subnets.

Reply:
Updated.